tests/db/test_bigchain_api.py

```diff
@@ -185,11 +185,11 @@ def test_get_previous_block_id(self, b):
 
     def test_create_new_block(self, b):
         new_block = b.create_block([])
-        block_hash = hash_data(util.serialize(new_block['block']))
+        block_hash = crypto.hash_data(util.serialize(new_block['block']))
 
         assert new_block['block']['voters'] == [b.me]
         assert new_block['block']['node_pubkey'] == b.me
-        assert PublicKey(b.me).verify(util.serialize(new_block['block']), new_block['signature']) is True
+        assert crypto.VerifyingKey(b.me).verify(util.serialize(new_block['block']), new_block['signature']) is True
```

Comment:
because you have to instantiate a key with a certain value first before you can verify

ofc you can create a wrapper function verify(msg, vk) but thats irrelevant I guess